packages/curve-ui-kit/src/shared/ui/DataTable/TableFiltersTitles.tsx

```diff
@@ -0,0 +1,12 @@
+import Typography from '@mui/material/Typography'
+
+export const TableFiltersTitles = ({ title, subtitle }: { title: string; subtitle?: string }) => (
+  <>
+    <Typography variant="headingSBold">{title}</Typography>
```

Comment:
bonus points if we can use opacity and transition to hide this slowly :nerd_face: